clients/python-wrapper/lakefs/branch.py

```diff
@@ -32,15 +32,22 @@ def create(self, source_reference_id: str | Reference, exist_ok: bool = False) -
         """
         Create a new branch in lakeFS from this object
 
+        Example of creating a new branch:
+
+        .. code-block:: python
+
+            import lakefs
+
+            branch = lakefs.repository("<repository_name>").branch("<branch_name>").create("<source_reference>")
+
         :param source_reference_id: The reference to create the branch from
         :param exist_ok: If False will throw an exception if a branch by this name already exists. Otherwise,
             return the existing branch without creating a new one
         :return: The lakeFS SDK object representing the branch
-        :raises:
-            NotFoundException if repo, branch or source reference id does not exist
-            ConflictException if branch already exists and exist_ok is False
-            NotAuthorizedException if user is not authorized to perform this operation
-            ServerException for any other errors
+        :raise NotFoundException: if repo, branch or source reference id does not exist
```

Comment:
not related to this pr - but I think we should use `repository` and not repo in non-code.

Reply:
Fixed thanks